onnxruntime/python/tools/transformers/fusion_gpt_attention_no_past.py

```diff
@@ -20,7 +20,7 @@ class FusionGptAttentionNoPast(Fusion):
     """
 
     def __init__(self, model: OnnxModel, num_heads: int):
-        super().__init__(model, "Attention", "LayerNormalization", "without past")
+        super().__init__(model, "Attention", ["LayerNormalization", "SkipLayerNormalization"], "without past")
```

Comment:
It seems like this fusion doesn't have tests yet as no tests broke when the SLN fusion was enabled for GPT2 by default. According to @tlwu, this is low priority as we use past states for the GPT models used in GreedySearch/Beamearch. FWIW- I updated this pattern as well to handle SLN.

The other two Attention fusions were updated and tests were updated to account for SLN